doc/cabal-package.rst

```diff
@@ -801,18 +800,45 @@ Library
 
     Build information for libraries.
 
-    A package can have one default library, which is identified by omitting the
-    ``name`` argument.
+    A package can include zero or more library components, being an optional
+    'main' unnamed library (which is identified by having no ``name`` argument)
+    and/or zero or more 'subsidiary' or 'ancilliary' named libraries (referred
+    to as 'sublibraries', which are identified by their ``name`` argument). A
+    sublibrary cannot have the same name as its package.
 
-    Starting with Cabal 2.0, sub-library components can be defined by setting
-    the ``name`` field to a name different from the current package's name; see
-    section on :ref:`Sublibraries <sublibs>` for more information. By
-    default, these sub-libraries are private and internal. Since Cabal 3.0,
-    these sub-libraries can also be exposed and used by other packages. See the
-    :pkg-field:`library:visibility` field and :ref:`Multiple Public Libraries
-    <publicsublibs>` for more information.
+    A sublibrary has a 'visibility' which can be 'public' or 'private' (the
+    default) - see the :pkg-field:`library:visibility` field. The 'main'
+    unnamed library is always public. A private sublibrary may be referred to as
+    an 'internal' library, because other packages cannot depend upon it.
 
-The library section should contain the following fields:
+    Before Cabal 3.0, all sublibraries were internal libraries. Before Cabal 2.0
+    a package could not include sublibraries.
+
+    A package being able to include more than one public library separates the
+    unit of distribution (the package) from the unit of buildable code (the
+    library). This is useful for Haskell projects with many libraries that are
+    distributed together as it avoids duplication and potential inconsistencies.
+
+    See :ref:`Sublibraries - Examples <sublibs>` for examples.
+
+A library section should contain the following fields:
+
+.. pkg-field:: visibility: visibility specifiers
+
+    :since: 3.0
+
+    :default:
+        ``private`` for sublibraries. Cannot be set for the 'main' unnamed
+        library, which is always public.
+
+    Can be set to ``public`` or ``private``. If set to ``public``, other
+    components of the package that provide the library and components of other
+    packages are able to depend on the library. If set to ``private``, only
+    other components of the package that provide it are able to depend upon it;
+    components of other packages are not able to depend upon it.
+
+    See the :pkg-field:`build-depends` field for the syntax to specify a 'main'
+    unnamed library or a sublibrary in a ``build-depends:``.
```

Comment:
```suggestion
    See the :pkg-field:`build-depends` field for the syntax to specify a dependency on a main library or a sublibrary.
```
As above I don't think we need to quote and called it unnamed. Also let's not say "build-depends" twice, rather let's say what it represents.

Reply:
I agree.